python/Ganga/Core/GangaRepository/Registry.py

```diff
@@ -746,137 +566,44 @@ def _write_access(self, _obj):
         Args:
             _obj (GangaObject): This is the object we want to get write access to and lock on disk
         """
-        logger.debug("_write_access")
-        obj = stripProxy(_obj)
-        obj_id = id(_obj)
-        if obj_id in self._inprogressDict:
-            return
-
-        with obj.const_lock:
-            self.__write_access(obj)
-
-    def __write_access(self, _obj):
-        """
-        This actually performs the file locks and gets access to the object
-        Args:
-            _obj (GangaObject): This is the object we want to get write access to and lock on disk
-        """
-        logger.debug("__write_acess")
-        obj = stripProxy(_obj)
-        this_id = id(obj)
-        if this_id in self._inprogressDict:
-            for this_d in self.changed_ids.itervalues():
-                this_d.add(self.find(obj))
-            return
+        obj = stripProxy(obj)
```

Comment:
We *shouldn't* need any of these `stripProxy` calls any more. It's worth checking them but I'm pretty sure they're now redundant.